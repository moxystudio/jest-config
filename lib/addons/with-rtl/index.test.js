'use strict';

const withRTL = require('.');

it('should match snapshot', () => {
    expect(withRTL()).toMatchSnapshot();
});
