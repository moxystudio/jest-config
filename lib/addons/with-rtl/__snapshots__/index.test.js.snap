// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
Object {
  "setupFilesAfterEnv": Array [
    "foo",
    "@testing-library/jest-dom/extend-expect",
  ],
}
`;
